packages/frontend/core/src/blocksuite/extensions/theme/preview-theme.ts

```
@@ -0,0 +1,73 @@
+    readonly disposables: (() => void)[] = [];
+
+    static override setup(di: Container) {
+      super.setup(di);
+      di.override(ThemeExtensionIdentifier, AffinePagePreviewThemeExtension, [
+        StdIdentifier,
+      ]);
+    }
+
+    constructor(std: BlockStdScope) {
+      super(std);
+      const theme$: Observable<ColorScheme> = framework
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)